templates/useroptions.pug

```diff
@@ -116,6 +116,7 @@ mixin us-chat
             option(value="never") Never
             option(value="onlyping") Only when I am mentioned or PMed
             option(value="always") Always
+      +rcheckbox("us-notifications", "Use notifications")
```

Comment:
I think desktop notifications should probably be handled the same way as the notification and page title options above -- it should be a tri-state (never / only when I am mentioned / always) instead of just on/off.

Reply:
Will do.